src/main/java/com/jmb/springfactory/controller/api/GroupTaskController.java

```diff
@@ -0,0 +1,45 @@
+package com.jmb.springfactory.controller.api;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.bo.QueryTaskObject;
+import com.jmb.springfactory.model.dto.TaskDto;
+import com.jmb.springfactory.service.task.TaskService;
+
+@RestController
+@RequestMapping("/group/{groupId}/task")
+public class GroupTaskController {
+
+    @Autowired
+    private TaskService taskService;
+
+    @PostMapping
+    public TaskDto create(@Valid @RequestBody TaskDto task, @PathVariable("groupId") Integer groupId)
+            throws NotFoundException, ServiceLayerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)